src/pathfinding_service/model/channel.py

```diff
@@ -8,7 +8,7 @@
 
 from pathfinding_service.constants import DEFAULT_REVEAL_TIMEOUT
 from pathfinding_service.exceptions import InvalidPFSFeeUpdate
-from raiden.exceptions import UndefinedMediationFee
+from raiden.tests.utils.mediation_fees import fee_receiver, fee_sender
```

Comment:
ok what I understand is that you take the fee calculation from the Raiden repo to do it in the pfs, now. Is that correct?

Reply:
Yes, exactly.